test/lit/passes/outlining.wast

```diff
@@ -180,25 +180,25 @@
 
   ;; CHECK:      (func $outline$ (type $0)
   ;; CHECK-NEXT:  (drop
-  ;; CHECK-NEXT:   (i32.add
-  ;; CHECK-NEXT:    (i32.const 0)
-  ;; CHECK-NEXT:    (i32.const 1)
+  ;; CHECK-NEXT:   (i32.sub
```

Comment:
This test changed because the order of the outlined functions to be created changed. This is theoretically possible now because we wait to add substrings to the result vector in hash-stringify-walker's removeOverlaps() until every interval for a repeat substring has been seen. For our purposes, It does not actually matter what order the substrings are in, because we create an OutliningSequence to represent each substring, and ensure that is sorted by idx, line 373 in Outlining.cpp.